test/battle/move_effect/ohko.c

```diff
@@ -20,6 +20,22 @@ SINGLE_BATTLE_TEST("Sheer Cold doesn't affect Ice-type Pokémon")
         MESSAGE("It doesn't affect the opposing Glalie…");
     }
 }
+
+SINGLE_BATTLE_TEST("OHKO moves can hit semi-invulnerable mons when the user has No-Guard")
```

Comment:
I would recommend moving this test into a `ability/no_guard.c` file, as this is a property of No Guard instead of OHKO move themselves.